src/components/Articles.jsx

```diff
@@ -0,0 +1,38 @@
+import ArticlePanel from "./ArticlePanel";
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+const Articles = ({ setArticles, articles }) => {
+  async function fetchArticles() {
+    const response = await axios.get(
+      `https://kangor.onrender.com/api/articles`
+    );
+    const artcleData = response.data;
+    return artcleData;
+  }
+  useEffect(() => {
+    fetchArticles().then((data) => {
+      setArticles(data);
+    });
+  }, []);
+  console.log(articles);
+  return (
+    <div className="all_item_articles">
+      {articles.map((article) => (
+        <ArticlePanel
+          key={article.item_id}
```

Comment:
instead of passing each property on the article down, you could just pass article down and then access all these properties that way.